site.json

```diff
@@ -30,7 +30,7 @@
     "index": {
       "startDate": "2024-01-30T17:30:00.000Z",
       "endDate": "2024-02-13T17:30:00.000Z",
```

Comment:
FYI We'll need to update before this time on Tuesday (either in the post-announcement or another delay post) otherwise the banner will disappear.